build.sh

```diff
@@ -0,0 +1,29 @@
+git config --global push.default simple
+git config --global user.email "travis"
+git config --global user.name "Travis CI"
+git remote rm origin
+git remote add origin git@github.com:fossasia/fossasia.org.git
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]
+then git clone git@github.com:fossasia/fossasia.org.git out
+cd out
+git fetch origin refs/pull/${TRAVIS_PULL_REQUEST}/merge:${TRAVIS_PULL_REQUEST}
```

Comment:
ya i made a new branch in my fork for this
I'll merge as soon as the ssh problem is fixed